src/vde_backend/pyToCassandra.py

```diff
@@ -280,6 +280,21 @@ def groupbyQuery(
 
 	return res_df
 
+
+def existTable(session, keyspace, table_name):
+	""" 
+	Check if a table exists in the cluster given a certain keyspace
+	"""
+	query = "SELECT table_name from system_schema.tables "
+	query += "where keyspace_name = '{}' ".format(keyspace)
+	query += "and table_name = '{}' ".format(table_name)
+	query += "ALLOW FILTERING;"
+
+	r = session.execute(query)
+
+	return (len(r.current_rows) > 0)

```

Comment:
Style remark: outer parentheses are not necessary, `len(r.current_rows) > 0` would work as well.